app/assets/stylesheets/card/_card.scss

```diff
@@ -0,0 +1,53 @@
+.card-container {
+    font-family: 'Fira Sans', sans-serif;
+    width: 600px;
+    height: 350px;
+    background-color: $color-lemon-light;
+}
+
+.logo {
+    font-size: 18px;
+    background-color: $color-lemon;
+    padding: 0.5em 1.75em;
+}
+
+.image-card {
+    display: grid;
+    grid-template-columns: 30% 1fr;
+    grid-column-gap: 25px;
+    grid-template-areas: 'a b';
+    margin: auto;
+    width: 90%;
+    padding-top: 1.4em;
+    padding-bottom: 1.4em;
+}
+
+.member-image {
+    grid-area: a;
+    img {
+        width: 150px;
```

Comment:
This is upscaling the image beyond its natural size so it's going to be blocky. For the purposes of this PR leave it as is but I do think we need to tackle that properly after this gets merged and figure out a pipeline for getting higher resolution images.